### PR TITLE
cbl to cob

### PR DESCRIPTION
**Pull request by axtens**

### CODE REVIEW

File: exercises/practice/hello-world/config.properties [hunk 1]
```diff
@@ -61,7 +61,7 @@ cobolcheck.test.program.path = ./
 # test program that contains the merged test code.
 # Example: For program ABCXYZ4, if suffix is T.CBL then the test program name will be ABCXYZ4T.CBL.
 #---------------------------------------------------------------------------------------------------------------------
-cobolcheck.test.program.name = TEST.CBL
```
Comment (KTSnowy) [inline exercises/practice/hello-world/config.properties:64]: Some of the lines have an uppercase .COB, would that cause any issues?

Comment (KTSnowy) [reply exercises/practice/hello-world/config.properties:64]: I selected the wrong line sorry, but the uppercase .COB is there.

Comment (KTSnowy) [reply exercises/practice/hello-world/config.properties:64]: It seems that the merge conflicts are because of the uppercase .COB?

**Comment by KTSnowy** [comment]
There we go, I fixed the merge conflicts

**Review by KTSnowy — APPROVED** [review]
> [KTSnowy](https://github.com/KTSnowy) self-requested a review

That's interesting, I didn't know you could self-request a review.

But everything looks good now, I think I fixed the merge conflicts

**Comment by axtens** [comment]
Cool! Thanks

**Pull request merged**